site/content/en/docs/tasks/run/multikueue/job.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Run Kubernetes Job in Multi-Cluster"
+linkTitle: "Kubernetes Job"
+weight: 2
+date: 2024-11-05
+description: >
+  Run a MultiKueue scheduled Kubernetes Job.
+---
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+For the ease of setup and use we recommend using at least Kueue v0.8.1 and Kubernetes v1.30.0.
+
+## Run Kubernetes Job example
+
+Once the setup is complete you can test it by running the [`job-sample.yaml`](/docs/tasks/run/jobs/#1-define-the-job) Job. 
+
+## Cluster with JobManagedBy enabled
+
+You can only set the `spec.managedBy` field on Jobs if you enable the JobManagedBy [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates) (disabled by default).
+The `JobManagedBy` feature gate is available in Kubernetes version at least 1.30.0 and disabled by default, it will become enabled by default since Kubernetes version 1.32.0.
+
+Check the [Delegation of managing a Job object to external controller](https://kubernetes.io/docs/concepts/workloads/controllers/job/#delegation-of-managing-a-job-object-to-external-controller) for more details.
+
+The batch/Job can work in two different ways depending on the state of `MultiKueueBatchJobWithManagedBy` feature gate, disabled by default.
+Check the [Change the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) for setup details.
+
+In this setup we recommend enabling also MultiKueueBatchJobWithManagedBy 
+
+When `MultiKueueBatchJobWithManagedBy` is enabled, Kueue defaults the `spec.managedBy` field to `kueue.x-k8s.io/multikueue`.
+This is the recommended option, which allows MultiKueue to send life updates reflecting the status of the mirror Job on the worker.
+
+{{% alert title="Note" color="primary" %}}
+Note: The MultiKueue admission check controller will `Reject` the workloads owned by a batch job that does not have the `spec.managedBy` field set to `kueue.x-k8s.io/multikueue`, causing them to be marked as `Finished` with an error indicating the cause.
+{{% /alert %}}
+
+When `MultiKueueBatchJobWithManagedBy` is disabled, the management cluster keeps the multikueue AdmissionCheckState `Pending` and therefore the Job suspended, during the remote job execution.
+The Job status sync happens only when the remote workload is marked as `Finished`.
+
+## Cluster with JobManagedBy disabled
+
+There is no requirement on Kubernetes version other than required for [Kueue installation](https://kueue.sigs.k8s.io/docs/installation/#before-you-begin).
+
+In this case `MultiKueueBatchJobWithManagedBy` should be disabled in Kueue. MultiKueue will not perform life updates of the mirror Job from a worker cluster.
```

Comment:
```suggestion
When `JobManagedBy` is enabled in your cluster you should make sure  `MultiKueueBatchJobWithManagedBy` 
is disabled in Kueue. This is important so that MultiKueue does not conflict with the build-in Job controller
on the management cluster. Note that Kueue will not perform life updates of the mirror Job from a worker 
cluster.
```